tests/js/spec/utils/dashboards/issueFieldRenderer.spec.tsx

```diff
@@ -133,4 +134,40 @@ describe('getIssueFieldRenderer', function () {
       expect(screen.getByText('Since issue began')).toBeInTheDocument();
     });
   });
+
+  it('can render annotations', async function () {
+    const renderer = getIssueFieldRenderer('annotations');
+
+    rtlMountWithTheme(
```

Comment:
we usually avoid renaming named exports. should we use only `mountWithTheme` here?

Reply:
Yup, updated! We've renamed `mountWithTheme` in the past due to some test files with existing enzyme tests to avoid clashing import names, but there's no enzyme here so we can undo the renaming